src/wasm_split_tools/emit.rs

```diff
@@ -492,8 +492,10 @@ impl<'a> ModuleEmitState<'a> {
         let indirect_table_size = self.emit_state.indirect_functions.table_entries.len() + 1;
         wasm_encoder::TableType {
             element_type: wasm_encoder::RefType::FUNCREF,
-            minimum: indirect_table_size as u32,
-            maximum: Some(indirect_table_size as u32),
+            minimum: indirect_table_size as u64,
+            maximum: Some(indirect_table_size as u64),
+            table64: false,
+            shared: false,
```

Comment:
@gbj I used the default values for these new fields in the new version. Please review them.